spec/exporters/monthly_district_report/diabetes/block_data_spec.rb

```diff
@@ -0,0 +1,136 @@
+require "rails_helper"
+
+def mock_block_repo(repo, district, month)
```

Comment:
Since this class directly calls the repo to get the data, it doesn't make sense to mock the repo here.
Also, even if the structure of the repo changes the tests will continue to pass.